### PR TITLE
Detect ChromeOS and treat it like Android

### PR DESCRIPTION
**Pull request by gesa**
On the long march to conquer the Chromebook soft keyboard bug, this PR updates the braintree/browser-detection package and makes sure ChromeOS inputs are handled like Android inputs.

### CODE REVIEW

**Review by hollabaq86 — APPROVED** [review]
👍

File: version [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```
Comment (crookedneighbor) [inline version]: Can we do this as part of a separate PR?

Probably needs a post publish script that also adds an `unreleased` section if we want to automate this fully.

**Review by crookedneighbor — CHANGES_REQUESTED** [review]
Can we take out the version script stuff? If we want to automate it, we should fully do it and do it as a separate PR, IMO.

Does setting a version script override the normal `npm version` task? Or does it occur before? or what? Any reason it can't be `preversion` instead?

(all questions for a separate PR, IMO)

**Comment by gesa** [comment]
@crookedneighbor fair enough. i had actually only added it in this PR bc i noticed in the log that historically the tag commit is the _same_ commit as the changelog. also this lib is tiny. i'll open a new PR with answers to your Qs.

**Review by crookedneighbor — APPROVED** [review]

**Pull request merged**